Hardcore.lua

```diff
@@ -2343,10 +2366,21 @@ function Hardcore:CHAT_MSG_ADDON(prefix, datastr, scope, sender)
 		end
 
 		if command == COMM_COMMANDS[10] then -- Received request for guild members
-			-- receiveDeathMsg(data, sender, command) would duplicate for sender
-			local commMessage = COMM_COMMANDS[11] .. COMM_COMMAND_DELIM .. data
-			CTL:SendAddonMessage("ALERT", COMM_NAME, commMessage, "GUILD")
-			return
+			local other_player_name = ""
+			local level = 0
+			local zone = ""
```

Comment:
rm; unused

Reply:
could we send the location as an id, and look up client side? we keep info, one less string to hack

Reply:
That sounds good